docs/aws/audit/cloudwatchmonitoring.mdx

```diff
@@ -9,7 +9,8 @@ sidebar_label: "Cloudwatch Audit"
 - [Authorization Failures Alarm](cloudwatchmonitoring/rules/authorization_failures_alarm)
 - [AWS Config Changes Alarm](cloudwatchmonitoring/rules/aws_config_changes_alarm)
 - [CloudTrail Changes Alarm](cloudwatchmonitoring/rules/cloudtrail_changes_alarm)
-- [CloudWatch Log Groups Should Be Encrypted](cloudwatchmonitoring/rules/cloudwatch_log_groups_not_encrypted)
+- [CloudWatch Log Group Emcrypted](cloudwatchmonitoring/rules/cloudwatch_log_groups_not_encrypted)
+- [CloudWatch Log Groups Should Be Encrypted With CMK](cloudwatchmonitoring/rules/cloudwatch_log_groups_with_cmk)
```

Comment:
did we not have this rule earlier?

Reply:
This policy document was not there, hence added it